pkg/vpcmodel/drawioGenerator.go

```diff
@@ -54,3 +55,29 @@ func (gen *DrawioGenerator) TreeNode(res DrawioResourceIntf) drawio.TreeNodeInte
 func (exn *ExternalNetwork) GenerateDrawioTreeNode(gen *DrawioGenerator) drawio.TreeNodeInterface {
 	return drawio.NewInternetTreeNode(gen.PublicNetwork(), exn.CidrStr)
 }
+
+func (g *groupedEndpointsElems) GenerateDrawioTreeNode(gen *DrawioGenerator) drawio.TreeNodeInterface {
+	// todo - need to implement, currently blocked in parse_args.go:
+	return gen.TreeNode((*g)[0])
+}
+
+func (g *groupedExternalNodes) GenerateDrawioTreeNode(gen *DrawioGenerator) drawio.TreeNodeInterface {
+	if len(*g) == 1 {
+		return gen.TreeNode((*g)[0])
+	}
+	tooltip := []string{}
+	for _, n := range *g {
+		tooltip = append(tooltip, n.(*ExternalNetwork).Cidr())
+	}
+	name := "multi cidr"
```

Comment:
Consider renaming to "various IP ranges" / "CIDR list" /... ?

Reply:
done